concent_api/concent_api/system_check.py

```diff
@@ -390,6 +390,22 @@ def create_error_55_use_signing_service_is_true_but_middleman_is_missing() -> Er
     )
 
 
+def create_error_56_if_additional_verification_call_time_is_not_set() -> Error:
```

Comment:
please remove "if" from the function name (it does fit to the naming convention used in other functions)

Reply:
:+1: done